Prolific/Prolific/Utils/Persistence/DAO.m

```diff
@@ -182,6 +182,20 @@ - (void)updateExistingSnippet:(Snippet *)snippet
     }];
 }
 
+- (void)updateAllSubmissionsForRound:(Round *)round
```

Comment:
This method isn't probably doing what you think. Let's dicuss it in the next 1 on 1